MANUAL_GCP_DEPLOYMENT.md

```diff
@@ -0,0 +1,18 @@
+# Deployment Guide for membrane-backend
```

Comment:
give a bit more material on what makes this deployment special

Reply:
I added the reasoning and benefits of mounting secrets as volume

Reply:
I don't understand, I explained the benefits of using secrets as volume is that not what I need to explain?

Reply:
this file is called MANUAL DEPLOYMENT GCP. It's not just about secrets.

Reply:
I think I am not advanced enough to understand what makes this deployment special because that was the big difference from the other deployments I made

Reply:
if you can specify so I can close please @rngadam

Reply:
@k-allagbe explained it's okay